src/dev/flang/tools/AceMode.java

```diff
@@ -0,0 +1,250 @@
+/*
+
+This file is part of the Fuzion language implementation.
+
+The Fuzion language implementation is free software: you can redistribute it
+and/or modify it under the terms of the GNU General Public License as published
+by the Free Software Foundation, version 3 of the License.
+
+The Fuzion language implementation is distributed in the hope that it will be
+useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+License for more details.
+
+You should have received a copy of the GNU General Public License along with The
+Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+*/
+
+/*-----------------------------------------------------------------------
+ *
+ * Tokiwa Software GmbH, Germany
+ *
+ * Source of class AceMode
+ *
+ *---------------------------------------------------------------------*/
+
+package dev.flang.tools;
+
+import dev.flang.parser.Lexer;
+import dev.flang.util.ANY;
+import java.util.stream.*;
```

Comment:
please do not use * in imports, in particular if all you need is
```suggestion
import java.util.stream.Stream;
```

Reply:
That's definitely better. I replaced the line in question by:
import java.util.stream.Collectors;
import java.util.stream.Stream;